dotnet/src/Connectors/Connectors.MistralAI/Client/MistralChatMessage.cs

```diff
@@ -10,22 +10,22 @@ namespace Microsoft.SemanticKernel.Connectors.MistralAI.Client;
 internal class MistralChatMessage
 {
     [JsonPropertyName("role")]
-    public string Role { get; set; }
+    public string? Role { get; set; }
 
     [JsonPropertyName("content")]
     public string Content { get; set; }
 
     /// <summary>
     /// Construct an instance of <see cref="MistralChatMessage"/>.
     /// </summary>
-    /// <param name="role">Must be one of: system, user, assistant</param>
+    /// <param name="role">If provided must be one of: system, user, assistant</param>
     /// <param name="content">Content of the chat message</param>
     [JsonConstructor]
-    internal MistralChatMessage(string role, string content)
+    internal MistralChatMessage(string? role, string content)
     {
-        if (role != "system" && role != "user" && role != "assistant")
+        if (role is not null && role != "system" && role != "user" && role != "assistant")
```

Comment:
```suggestion
        if (role is not null && role is not "system" or "user" or "assistant")
```

Reply:
That didn't work but I did clean this up